go.sum

```diff
@@ -722,7 +738,7 @@ github.com/openshift/library-go v0.0.0-20200327125526-163b2f0d6264 h1:VcZjTupwjx
 github.com/openshift/library-go v0.0.0-20200327125526-163b2f0d6264/go.mod h1:CfydoH0B+RYs22uQZQ36A1mz5m5zhucpMGh8t5s71v4=
 github.com/openshift/moby-moby v1.4.2-0.20190308215630-da810a85109d h1:1LuQzDKgiXj1omPNDcY1E/mEOE/90jdobR+7WBfBQYA=
 github.com/openshift/moby-moby v1.4.2-0.20190308215630-da810a85109d/go.mod h1:LJM49W8fBVSj+rvcopJZu9mgH5Tx6HwLHySIYeGeu4k=
-github.com/openshift/source-to-image v1.3.0 h1:YDEH0PY0sMaUNbpJAGnDZA94tkAD6qi1Rou8ycWRPRc=
+github.com/openshift/source-to-image v1.3.0 h1:ZktIgJ85Vn0HOyHdS6uQOMXhmYwc5h76w/6LpnJ60L8=
```

Comment:
@adambkaplan that's a strange behavior on working with `go mod` for this PR, I had to remove `source-to-image` line from `go.sum`, in order to match the correct commit checksum.

Reply:
think this is because I merged a few doc changes into s2i and moved the tag. No golang code changed.

Reply:
Thanks.